src/includes/set-release/xamarin.mdx

```diff
@@ -0,0 +1,10 @@
+```csharp {tabTitle:C#}
+using Sentry;
```

Comment:
Isn't this `Sentry.Xamarin`?

Reply:
we are using namespace Sentry for the public classes.